README.md

```diff
@@ -1 +1,27 @@
-# jv-dao-service
\ No newline at end of file
+# HW 03
+- Create class `Driver`.
+```java
+public class Driver {
+    private Long id;
+    private String name;
+    private String licenseNumber;
+}
+```
+- Create `ManufacturerService` and its implementation. Annotation `@Service` has already been given to you.
+### ManufacturerService methods:
+    - Manufacturer create(Manufacturer manufacturer);
+    - Manufacturer get(Long id);
+    - List<Manufacturer> getAll();
+    - Manufacturer update(Manufacturer manufacturer);
+    - boolean delete(Long id);
+- Create Dao and Service layers for `Driver` and theirs implementations.
+### DriverService methods:
+    - Driver create(Driver driver);
+    - Driver get(Long id);
+    - List<Driver> getAll();
+    - Driver update(Driver driver);
+    - boolean delete(Long id);
+
+- Test your code in `main`.
```

Comment:
let's add that they should put `@Service` annotation above the service implementations

and that they should test only service methods in Main, not dao